docs/recipes/endpoint-testing-with-mongoose.md

```diff
@@ -156,3 +139,13 @@ And you're done!
 You may choose to abstract code for `test.before`, `test.beforeEach`, `test.afterEach.always` and `test.after.always` into a separate file.
 
 To see a demo of this configuration file, look at https://github.com/zellwk/ava-mdb-test
+
+## Why `test.serial` instead of `test`
```

Comment:
See `User` and `UserOnServer2` models constructed on one schema with different connections in the following example:
https://github.com/nodkz/mongodb-memory-server#several-mongoose-connections-simultaneously